app/controllers/tweet_controller.rb

```diff
@@ -0,0 +1,84 @@
+require 'pry'
+
+class TweetController < ApplicationController
+
+  get '/tweets' do
+
+    if !logged_in?
+      redirect to '/login'
+    end
+    @user = User.find_by_id(session[:user_id])
```

Comment:
Why aren't we loading our tweets into an instance variable here.

example: 

```
@tweets = Tweet.all
```
we should be separating our logic using the MVC design patter. We should not be making database calls in the view.